modules/dnds/1.2/src/R/run_dnds.R

```diff
@@ -27,6 +27,18 @@ maf = read.table(
 
 maf = as.data.frame(maf)
 
+if (grepl("38", maf$NCBI_Build[1])) {
+    cat("Will run using the hg38 reference.\n")
+    load(snakemake@params[[3]]) # This will load the RefCDS from the path at config
```

Comment:
Isn't there a way to use named parameters here instead of referring to them positionally?

Reply:
yep, there is - was just going with a consistent style to match other spots in this script where we refer to the snakemake objects but will swicth!